department-of-veterans-affairs/caseflow
# Separated out logic for scheduling veteran hearings

Resolves #15029

Stacked PR:
- #15117 - Jest Tests

### Description
Adds a new component to wrap the ScheduleVeteranForm and manage the state. Will eventually fully replace the logic for the AssignHearingModal component

### Acceptance Criteria

### Testing Plan
1. Run storybook locally: make storybook
1. Navigate to the schedule veteran component Hearings/Components/Schedule Veteran
1. Ensure that you are able to change the form and it looks according to the mockups

### Storybook Story
*For Frontend (Presentationa) Components*
